samples/deployment/kvm/deploy_vm.yaml

```diff
@@ -0,0 +1,49 @@
+- name: "Deploy vFTD at the KVM host"
```

Comment:
I guess, it should be `Deploy on` or `Deploy to`.
https://english.stackexchange.com/questions/5756/deployed-to-vs-deployed-on/5761

Reply:
fixed